doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -48,18 +49,42 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
+| `req.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management tools system at both HW device level (embedded) and HW platform level (external to device). |
+| `req.inf.com.03` | Compute | The Architecture **should** support symmetrical CPU multi-processing with shared memory access as well as multi-threading. |
+| `req.inf.com.04` | Compute | The Architecture **must** support Multiple CPU SKU options to support various infrastructure profiles (Base, Network Intensive, and Compute Intensive).
+| `req.inf.com.05` | Compute | The Architecture **must** support Multiple memory type, channel, socket and speed options regarding performance and capacity to support various infrastructure profiles (Base, Network Intensive, and Compute Intensive).
+| `req.inf.com.06` | Compute | The Architecture **must** support Hardware Platforms with NUMA capabilities.
+| `req.inf.com.07` | Compute | The Architecture **must** support CPU Pinning.
+| `req.inf.stg.01` | Storage | The Architecture **must** provide storage solution for VM Instances to support Block/Image and local VNF File system storage. |
 | `req.inf.stg.02` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports Block storage, object storage and flat files. |
-| `req.inf.ntw.01` | Network | The Architecture **must** include an SDN integration to support provisioning of network services from host-based OpenStack Neutron networking VTEPs to the Border Edge based VRFs. |
-| `req.inf.ntw.02` | Network | The Architecture **must** Localize intra-host traffic to provide low latency, high throughput, and resiliency. |
-| `req.inf.ntw.03` | Network | The Architecture **shall** support service function chaining from Data plane and fabric perspective. |
-| `req.inf.ntw.04` | Network | The Architecture **must** allow for East/West tenant traffic within the cloud (via tunnelled encapsulation overlay - VXLAN or Geneve). |
-| `req.inf.ntw.05` | Network | The Architecture **shall** support Distributed Virtual Routing (DVR) to allow compute nodes to route traffic efficiently |
-| `req.inf.acc.01` | Acceleration | The Architecture **shall** support Application Specific Acceleration (exposed to VNFs) xxx. |
-| `req.inf.acc.02` | Acceleration | The Architecture **shall** support NFVI Acceleration (such as SmartNICs) |
-| `req.inf.acc.03` | Acceleration | The Architecture **shall not** rely on SR-IOV PCI-Pass through to provide acceleration to VNFs |
+| `req.inf.stg.03` | Storage | The Architecture **should** be able to accommodate VNFs that store back into its image through use of hypervisor attached volumes. |
+| `req.inf.stg.04` | Storage | The Architecture **should** make the immutable images available via location independent means. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable VIM instance storage |
```

Comment:
What is meant with "VIM instance storage"? And: do we have a metric definition for "high-performance" storage we can refer to?

Reply:
VIM instance storage is storage created for a VIM instance. WE can provide metric, however HSS (high speed storage) or high-performance storage is a well known industry term. Check Google!

Reply:
Do you mean really VIM instance or VM instance? Ok, if there is a common understanding of HSS, but we have defined metrics for "[.gold storage extension](https://cntt-n.github.io/CNTT/doc/ref_model/chapters/chapter04.html#4.2.3)" in the Reference Model. Why not using it?